src/components/Thought/Thought.css

```diff
@@ -0,0 +1,60 @@
+.thought-container {
+    width: auto;
+    border: 1px solid black;
+    box-shadow: 5px 5px 0 0 black;
+    box-sizing: border-box;
+    padding: 20px;
+    margin: 15px 0;
+    margin-left: 25%;
+    margin-right:25%;
+   
+}
+
+.thought-info {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 15px;
+
+}
+
+.like-button {
+    animation: heartbeat 2s infinite cubic-bezier(0.4, 0, 0.6, 1.5);
+    background-color: lightgrey;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;   
+    border: none;
+    font-size: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+}
```

Comment:
Looks good, but might need some margin or something that doesn't make it go over the like count.
Also maybe add a state that tracks if the button has been clicked so that it changes color.